docs/use-cases/sunhwi_use_case.md

```diff
@@ -1,5 +1,15 @@
-## use-case for post measurement
-* `Save` takes current measurement save to a local location with a user-typed file name.
-* `Read` recalls previously saved data and show on GUI.
-* `Analyze` enables a fitting based on the active data set and shows the goodness of the fit.
-	* `Save analysis` saves new data that contains both raw data and analyzed data.
\ No newline at end of file
+## use case for post measurement - save
+Below cases are intended for users who completed one of following measurements on pytentiostat: cyclic voltammetry, linear sweep voltammetry, or chronoamperometry.
+
+* Check on the user-interface that the measurement is currently complete.
```

Comment:
Thanks Sun Hwi,

Here is your UC in compliant form (to the best of my understanding of what your intent was):

1. pytentiostat returns status of the experiment in the form "in-progress", "succeeded", "failed", "interrupted"
1. Experimenter Checks for run status on the user-interface.

On successful termination,  
1. pytentiostat requests filename and location from experimenter, offering reasonable defaults
1. experimenter inputs local the local file location and file name.

not sure if it the user or pytentiostat doing this.....
* Verify the saved file by opening on a spreadsheet program and compare it with data on the user-interface